modules/globebrowsing/src/tileprovider/tileproviderbyindex.cpp

```diff
@@ -67,21 +85,30 @@ documentation::Documentation TileProviderByIndex::Documentation() {
 TileProviderByIndex::TileProviderByIndex(const ghoul::Dictionary& dictionary) {
     ZoneScoped;
 
-    const Parameters p = codegen::bake<Parameters>(dictionary);
+    Parameters p = codegen::bake<Parameters>(dictionary);
+
+    // For now we need to impute the LayerGroupID this way. We don't want it to be part of
```

Comment:
You have a great vocabulary! "impute" 😮 Maybe use a synonym, like represent or assigne? I had to google 😅

Reply:
But it's such a great word :'(
![image](https://github.com/OpenSpace/OpenSpace/assets/4921724/8bf5bfd8-9f83-4b5a-9841-ec02c4b044fc)

I changed it to "inject"